README.md

```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
+     
+      `composer install` 
+     
+      and then run this command:
+      
+      `vendor/bin/phpunit Tests/Unit/EmogrifierTest.php`
+      
+      If you want to uninstall the development packages that were installed when you ran `composer install`, you can 
```

Comment:
Could you please change the (new) Markdown lines to be 80 characters or less? This will avoid wrapping on GitHub.